app/Enum/DeleteStatus.php

```diff
@@ -9,4 +9,5 @@ enum DeleteStatus: int
     case NOT_DELETED = 0;
     case SOFT_DELETE = 1;
     case HARD_DELETE = 9;
+    case PARMANENT_DELETE = 10;
```

Comment:
Why this permanent delete ?
Isn't the hard delete enough ?

Reply:
Vai hard delete dile ki parmanently delete hobe?
Akhon akjon user blood need post diye dice, kinto tar mone halo kono karone post er dorker nai, tai post ta delete diye dibe.

Reply:
I think, How we would proceed like, if somethings needed to be deleted, that will be first updated as hard delete.

Then in next 30 days or every day we'll run a cron job which will remove all data where permanent_delete true.